ts/screens/preferences/OldServicesHomeScreen.tsx

```diff
@@ -131,6 +286,8 @@ class OldServicesHomeScreen extends React.Component<Props> {
               isExperimentalFeaturesEnabled={
                 this.props.isExperimentalFeaturesEnabled
               }
+              onLongPressItem={this.handleOnLongPressItem}
```

Comment:
This is a little confusing because we are passing handlers to the component that doesn't need it.
In place of using these three params:
```typescript
  onLongPressItem: () => void;
  isLongPressEnabled: boolean;
  onSwitch: (service: ServicePublic) => void;
```
you could avoid boolean and make the handlers optional
  ```typescript
onLongPressItem?: () => void;
onSwitch?: (service: ServicePublic) => void;
```
so if a component wants to use the long-press feature on a _ServicListItem_ (New or Old) it has to declare the handlers.

so, for example, in ServiceListItem in place of this
```typescript
{isLongPressModeEnabled ? (
            <Col size={2}>
              <Switch
                key={`switch-service-${profileVersion}`}
                value={pot.isSome(enabledChannels)}
                onValueChange={() => true}
              />
            </Col>
          ) : (
            <IconFont name="io-right" color={variables.brandPrimary} />
          )}
```
we could have 
```typescript
{onLongPress ? (
            <Col size={2}>
              <Switch
                key={`switch-service-${profileVersion}`}
                value={pot.isSome(enabledChannels)}
                onValueChange={() => true}
              />
            </Col>
          ) : (
            <IconFont name="io-right" color={variables.brandPrimary} />
          )}
```

Reply:
Ok, I should have fixed it